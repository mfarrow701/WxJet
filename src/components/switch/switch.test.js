import React from 'react';
import {shallow} from 'enzyme';
import Switch from './switch';

describe('Search component', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<Switch />);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the search component after load', () => {
        expect(component).toMatchSnapshot();
    });
});
