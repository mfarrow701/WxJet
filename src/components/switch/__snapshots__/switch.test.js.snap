// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render the search component after load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Switch />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <input
          checked={undefined}
          onChange={undefined}
          type="checkbox"
/>,
        <span
          className="Slider"
/>,
      ],
      "className": "Switch",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "checked": undefined,
          "onChange": undefined,
          "type": "checkbox",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "Slider",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
    ],
    "type": "label",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <input
            checked={undefined}
            onChange={undefined}
            type="checkbox"
/>,
          <span
            className="Slider"
/>,
        ],
        "className": "Switch",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "checked": undefined,
            "onChange": undefined,
            "type": "checkbox",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "Slider",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
      ],
      "type": "label",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
